AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -100,70 +100,70 @@ public final class PayToComponent: PaymentComponent,
 
     /// The payment flow selection title  label item.
     internal lazy var flowSelectionTitleItem: FormLabelItem = {
-        itemsProvider.flowSelectionTitleItem
+        itemsProvider.flowSelectionTitleItem()
     }()
 
     /// The segment control item to choose the payTo flow.
     internal lazy var flowSelectionItem: FormSegmentedControlItem = {
-        let item = itemsProvider.flowSelectionItem
+        let item = itemsProvider.flowSelectionItem()
         item.selectionHandler = { [weak self] in
             self?.didChangeSegment($0)
         }
         return item
     }()
     
     internal lazy var phoneNumberItem: FormPhoneNumberItem = {
-        itemsProvider.phoneNumberItem
+        itemsProvider.phoneNumberItem()
     }()
 
     /// The  account holder firstname text input item.
     internal lazy var firstNameInputItem: FormTextInputItem = {
-        itemsProvider.firstNameInputItem
+        itemsProvider.firstNameInputItem()
     }()
 
     /// The  account holder lastname text input item.
     internal lazy var lastNameInputItem: FormTextInputItem = {
-        itemsProvider.lastNameInputItem
+        itemsProvider.lastNameInputItem()
```

Comment:
Maybe to make it more explicit you could name it something like

`itemsProvider.provideLastNameInputItem()`
`itemsProvider.buildLastNameInputItem()`
`itemsProvider.assembleLastNameInputItem()`

Basically just a verb that indicates that it's generating an item instead of accessing one 🤷‍♂️

Reply:
Agreed, otherwise we hide it inside and it might be improtant to know we get a new instance every time we access it.
Another name option is `makeLastNameInputItem()`

Reply:
Alright, in that case, the keyword we seem to use most seems to be `create`. We good with that one?

Reply:
Sure, if it is consistent with such cases in the existing codebase